src/styles/blocks/container.scss

```diff
@@ -0,0 +1,44 @@
+.container {
+  height: 600px;
+  display: grid;
+
+  @media (max-width: 600px) {
+    & {
+      grid-template-columns: 1fr;
+      grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
```suggestion
      grid-template-rows: repeat(6, 1fr);
```

Reply:
we could use repeat and we could save some time from hardcoding! you can do the same with the other values removing the 1fr format for the repeat format.